GpxLocationManager/GpxParser.swift

```diff
@@ -24,19 +24,19 @@ open class GpxParser: NSObject, XMLParserDelegate {
   private static let dateFormat = "yyyy-MM-dd'T'HH:mm:ss'Z'"
   private static let accuracy: CLLocationAccuracy = 5.0
   private enum ParsingState: String {
-    case Trkpt = "trkpt"
-    case Name = "name"
-    case Ele = "ele"
-    case Time = "time"
```

Comment:
I love that SwiftLint catches this sort of no-no. As an aside, I agree that `trkpt` is a terrible name, but I got it from the GPX standard. I do agree that `trackpoint` is better.